lib/linter-glsl.js

```diff
@@ -85,13 +87,15 @@ const parseGlslValidatorResponse = (inputs, output) => new Promise((resolve) =>
       const linkMatch = new RegExp(linkRegex.replace('{{typeName}}', typeName)).exec(line);
       if (linkMatch) {
         toReturn.push({
-          type: linkMatch[1],
-          text: linkMatch[2].trim(),
-          filePath: shader.fullFilename,
-          range: [
-            [0, 0],
-            [0, 0],
-          ],
+          severity: linkMatch[1].toLowerCase(),
+          excerpt: linkMatch[2].trim(),
+          location: {
+            file: shader.fullFilename,
+            position: [
```

Comment:
This should use `helpers.generateRange(textEditor, 0)` to highlight the entire first row. The current range of `[[0, 0], [0, 0]]` has _no_ visible indicator in the UI.

(The `TextEditor` reference will need to be passed in.)